src/@chakra-ui/gatsby-plugin/components/Button.ts

```diff
@@ -70,5 +70,19 @@ export const Button: ComponentStyleConfig = {
       color: "primary",
       borderColor: "primary",
     },
+    icon: {
+      appearance: "none",
+      display: "grid",
+      placeItems: "center",
```

Comment:
The button content is centered by default. If you inspect the [Button](https://chakra-ui.com/docs/components/button/usage) you will see that it is flex displayed and is centered.
```suggestion
```